From 1973aae844254e1e186b8bd2d93c91927c20c53d Mon Sep 17 00:00:00 2001
From: "Tao, Jing" <jing.tao@intel.com>
Date: Thu, 11 Dec 2014 13:28:33 +0800
Subject: [PATCH 193/196] ov5693-ecs: delete ov5693_1296x736_30fps setting

The 1296x736 setting is not full FOV setting so it is useless.

Change-Id: Id16af8c12d8879f5d947fc68eabed10a913ad57a
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-3965
Signed-off-by: Tao, Jing <jing.tao@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/32354
Reviewed-by: HaiqiangX Yao <haiqiangx.yao@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
---
 .../camera/drivers/media/i2c/ov5693-ecs/ov5693.h           | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
index dfee36a..8b98828 100755
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
@@ -912,20 +912,6 @@ struct ov5693_resolution ov5693_res_video[] = {
 		.regs = ov5693_736x496,
 	},
 	{
-		.desc = "ov5693_1296x736_30fps",
-		.width = 1296,
-		.height = 736,
-		.fps = 30,
-		.pix_clk_freq = 160,
-		.used = 0,
-		.pixels_per_line = 3504,
-		.lines_per_frame = 1522,
-		.bin_factor_x = 2,
-		.bin_factor_y = 2,
-		.bin_mode = 1,
-		.regs = ov5693_1296x736,
-	},
-	{
 		.desc = "ov5693_1636P_30fps",
 		.width = 1636,
 		.height = 1096,
-- 
1.9.1

